### PR TITLE
docs(ja): update additional-aws-resources page

### PR DESCRIPTION
**Pull request by g-grass**
This PR is a Japanese version of Additional AWS Resources page.
Added Customizing the Parameters section.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

**Comment by efekarakus** [comment]
Thank you @g-grass for the pull-request 😍   
Just to provide some transparency: @toricls is currently on vacation, but once he comes and approves the PR it should automatically get merged.

File: site/content/docs/developing/additional-aws-resources.ja.md [hunk 1]
```diff
@@ -117,3 +117,36 @@ Outputs:
     
     * `addons/` ディレクトリで定義するポリシーでは[最小限のアクセス権を付与する](https://docs.aws.amazon.com/ja_jp/IAM/latest/UserGuide/best-practices.html#grant-least-privilege)
     * `addons/` ディレクトリで定義したリソースに対してのみアクセスできるようにポリシーを制限するために [追加セキュリティに対するポリシー条件を使用する](https://docs.aws.amazon.com/ja_jp/IAM/latest/UserGuide/best-practices.html#use-policy-conditions) 
+
+
+### `Parameters` セクションのカスタマイズ
+
+Copilot では、 `App`, `Env` そして `Name` [パラメーター](https://docs.aws.amazon.com/ja_jp/AWSCloudFormation/latest/UserGuide/parameters-section-structure.html)がテンプレートに定義されている必要があります。もし、 Service スタック内のリソースを参照する[パラメーター](https://docs.aws.amazon.com/ja_jp/AWSCloudFormation/latest/UserGuide/parameters-section-structure.html)を追加したい場合は、 `addons.parameters.yml` ファイルを作成します。
```
Comment (toricls) [inline site/content/docs/developing/additional-aws-resources.ja.md]: Can we update this sentence like this? (I think I understand your **awesome** intent here, but let me know if I'm not ‪🤗‬)

```suggestion
Copilot では、 `App`, `Env` そして `Name` パラメーターがテンプレートに定義されている必要があります。もし、 Service スタック内のリソースを参照するを追加したい場合は、 `addons.parameters.yml` ファイルを作成します。テンプレート内パラメーターのデータ型やデフォルト値の指定方法については、[CloudFormation テンプレートでのパラメーター定義](https://docs.aws.amazon.com/ja_jp/AWSCloudFormation/latest/UserGuide/parameters-section-structure.html)に準じます。
```

File: site/content/docs/developing/additional-aws-resources.ja.md [hunk 2]
```diff
@@ -117,3 +117,36 @@ Outputs:
...
+
+```term
+.
+└── addons/
+    ├── template.yml
+    └── addons.parameters.yml # このファイルをaddons/ディレクトリの下に追加します
```
Comment (toricls) [inline site/content/docs/developing/additional-aws-resources.ja.md]:
```suggestion
    └── addons.parameters.yml # このファイルを addons/ ディレクトリの下に追加します
```

**Comment by g-grass** [comment]
Hi @efekarakus
Thank you for your comment!
It's totally fine. Have a nice vacation😀

File: site/content/docs/developing/additional-aws-resources.ja.md [hunk 3]
```diff
@@ -117,3 +117,36 @@ Outputs:
...
+```
+
+以下のように、 `addons.parameters.yml` にワークロードスタックから参照する[パラメーター](https://docs.aws.amazon.com/ja_jp/AWSCloudFormation/latest/UserGuide/parameters-section-structure.html)を定義することができます。
```
Comment (toricls) [inline site/content/docs/developing/additional-aws-resources.ja.md]:
```suggestion
`addons.parameters.yml` には、Service スタック内の値を参照するパラメーターを例えば以下のように定義できます。
```

File: site/content/docs/developing/additional-aws-resources.ja.md [hunk 4]
```diff
@@ -117,3 +117,36 @@ Outputs:
...
+```yaml
+Parameters:
+  ServiceName: !GetAtt Service.Name
+```
+最後に、新しい[パラメーター](https://docs.aws.amazon.com/ja_jp/AWSCloudFormation/latest/UserGuide/parameters-section-structure.html)を参照するようにテンプレートファイルを更新してください。
```
Comment (toricls) [inline site/content/docs/developing/additional-aws-resources.ja.md]:
```suggestion
最後に、新しいパラメーターを参照するようにテンプレートファイルを更新しましょう。
```

File: site/content/docs/developing/additional-aws-resources.ja.md [hunk 5]
```diff
@@ -117,3 +117,36 @@ Outputs:
...
+
+```yaml
+Parameters:
+  # AWS Copilotで必要なパラメーター
```
Comment (toricls) [inline site/content/docs/developing/additional-aws-resources.ja.md]:
```suggestion
  # AWS Copilot で必要なパラメーター
```

File: site/content/docs/developing/additional-aws-resources.ja.md [hunk 6]
```diff
@@ -117,3 +117,36 @@ Outputs:
...
+  App:
+    Type: String
+  Env:
+    Type: String
+  Name:
+    Type: String
+  # addons.parameters.ymlで追加したパラメーター
```
Comment (toricls) [inline site/content/docs/developing/additional-aws-resources.ja.md]:
```suggestion
  # addons.parameters.yml で追加したパラメーター
```

**Review by toricls — CHANGES_REQUESTED** [review]
Thank you so much @g-grass! Left some (nit) comments ‪🤗‬

File: site/content/docs/developing/additional-aws-resources.ja.md (hunk 1 above)
Comment (g-grass) [reply site/content/docs/developing/additional-aws-resources.ja.md]: @toricls 
Thanks for the suggestion!  I don't think there is any particular difference in perception.
It is easier for me to understand what you are suggesting.

File: site/content/docs/developing/additional-aws-resources.ja.md (hunk 2 above)
Comment (g-grass) [reply site/content/docs/developing/additional-aws-resources.ja.md]: Thank you. I overlooked it. 🙇️  
(including other parts).

**Comment by g-grass** [comment]
@toricls 
Thank you for the review! 
I committed all your suggestions.

**Review by toricls — APPROVED** [review]
Looks perfect to me ✨ Thank you @g-grass!

**Review by Lou1415926 — APPROVED** [review]
Thank you very much for your contribution ❤️

**Pull request merged**